runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -867,15 +902,15 @@ ClientSessionData::getClassRecord(J9Class *clazz, JITServer::ServerStream *strea
          );
```

Comment:
Losing error information here because `missingLoaderInfo` was never set to true.